source/help/getting-started/signing-in.rst

```diff
@@ -4,8 +4,9 @@ Signing in
 --------------
 
 You can sign in to your team from the root page of your Mattermost site
-``https://domain.com``. After signing in, you will be able to view a
-list of your teams and select which one to open.
+``https://domain.com``. After signing in, the team that appears first on your
```

Comment:
You should use example.com, which is a special domain set aside for use in documentation. Domain.com is an actual, existing, website.

Also, the way it's worded here, it sounds like you sign in to Mattermost by going to https://domain.com